Libraries/StyleSheet/processTransform.js

```diff
@@ -26,7 +26,12 @@ var stringifySafe = require('stringifySafe');
  * interface to native code.
  */
 function processTransform(transform: Object): Object {
-  var result = MatrixMath.createIdentityMatrix();
+  // Android implementation of transform property accepts the list of transform
+  // properties as opposed to a transform Matrix. This is necessary to control
+  // transform property updates completely on the native thread.
+  if (Platform.OS === 'android') {
```

Comment:
Yea I think we should fix this before shipping this to avoid regressing on dev experience.